src/databricks/labs/remorph/mixins/root_table.py

```diff
@@ -0,0 +1,143 @@
+import logging
```

Comment:
probably rename this file to `root_tables` (plural)

Reply:
Done